components/brave_ads/core/internal/common/test/ads_client_notifier_waiter.cc

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
```

Comment:
Would be great if this was located with ads_client_notififier.cc

Reply:
Fixed